### ISSUE TITLE
Duda sobre HP actual y HP máximo

### ISSUE THREAD

**Issue by Martinviald** [franciscogazitua/IIC2113-2024#183]
Hola! En el enunciado se menciona que las unidades tienen los atributos HP máximo y HP actual como dos atributos distintos, pero en el archivo json solo se nos entrega un único HP. ¿Debemos considerar ese HP inicialmente entregado como el HP máximo? o si no ¿cómo determinamos el HP máximo?

**Comment by RodrigoToroIcarte** [comment]
Sí, el HP del json es el HP máximo de la unidad. Las unidades comienzan con vida máxima :)
